arch/aarch64/mmu.c

```diff
@@ -2,7 +2,16 @@
 
 #include <kernel/errno.h>
 
-phys_t virt_to_phys(virt_t addr)
+
+/// \file          arch:aarch64:mmu.c
+/// \brief         The virt_to_phys function converts address from virtual to
+///                physical memory
+///
+/// \param    addr Virtual address to translate
+/// \returns       returns physical address of addr
```

Comment:
This is not exactly the same format, which is already in use.

Specifically, the `\brief` tag is not needed. The example I wanted you to look at looks as follows:

```c
/// Returns `true` if local interrupts are disabled; otherwise returns `false`.
///
/// \param flags The interrupt mask.
/// \return `true` if local interrupts are disabled; otherwise returns `false`.
static inline bool arch_local_interrupt_is_disabled(unsigned long flags)
```

In this case, the comment ought to look as follows:

```c
/// Converts a virtual memory address to physical address.
///
/// \param addr Virtual address to translate
/// \returns physical memory address of \addr
phys_t virt_to_phys(virt_t addr)
```

Reply:
I can add like how you have, not a problem; it's in best tips having [`\brief` tag](http://www.doxygen.nl/manual/commands.html#cmdbrief) and proper tags when adding  Doxygen documentation. I have no strong opinions though, just letting you think once more of what you are saying.

Reply:
I just read the Gitter ping.